code/modules/antagonists/valentines/heartbreaker.dm

```diff
@@ -3,6 +3,7 @@
 	roundend_category = "valentines"
 	show_in_antagpanel = FALSE
 	show_name_in_check_antagonists = TRUE
+	soft_antag = TRUE
```

Comment:
Current adminbus discussion says that heartbreaker can act as a traitor, so this should probably count for the antag count.